DESCRIPTION

```diff
@@ -0,0 +1,15 @@
+Package: HUCAgg
+Type: Package
+Title: Functions to aggregate WBD HUC12s
+Version: 0.0
+Date: 2015-10-06
+Author: David Blodgett
+Maintainer: David Blodgett <dblodgett@usgs.gov>
+Depends:
+    maptools, sp, rgeos
```

Comment:
Imports instead of Depends? 

Imports I think is preferred here because you really only want to import certain functions or classes from those packages